documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -117,17 +117,17 @@ and ``using namespace sycl;`` directives when referring to policy classes and fu
 
 .. code:: cpp
 
-   auto policy_a = device_policy<class PolicyA> {};
+   auto policy_a = device_policy<class PolicyA>();
```

Comment:
I would rather keep curly braces for variable initializations, as it clearly distinguishes these from function declarations and calls.

Reply:
Fixed